en/install-openstack.md

```diff
@@ -0,0 +1,580 @@
+Title: Install OpenStack
+TODO: Add image of deployment from the Juju GUI
+      Add link to the bundle file
+
+# Install OpenStack
+
+Now that we've installed and configured [MAAS][installmaas] and successfully
+deployed a [Juju][installjuju] controller, it's time to do some real work; use
+Juju to deploy [OpenStack][openstack], the leading open cloud platform. 
+
+We have two options when installing OpenStack. 
+
+1. Install and configure each OpenStack component separately. Adding
+   Ceph, Compute, Swift, RabbitMQ, Keystone and Neutron in this way allows you
+   to see exactly what Juju and MAAS are doing, and consequently, gives you a
+   better understanding of the underlying OpenStack deployment. 
+1. Use a [`bundle`][bundle] to deploy OpenStack with a single command.  A
+   bundle is an encapsulation of a working deployment, including all
+   configuration, resources and references. It allows you to effortlessly recreate
+   or share from deployment with a single command.
```

Comment:
'share from deployment' ? I mean, I know what you are saying but the wording is a bit jarring